### ISSUE TITLE
Time node triggers even disabled

### ISSUE THREAD

**Issue by DirkMu** [zachowj/node-red-contrib-home-assistant-websocket#405]
### Describe the bug

This is a follow-up issue to #404 
I am using several time nodes which are exposed to HA. After a HA restart the switch for the time node becomes unavailable. Thus, I re-deploy the time node by changing the name in NR. The entity in HA becomes available and since the switch is turned off the node is DISABLED. However, the node in NR will trigger once. The time node itself uses the time from a input_datetime in HA. Don't know if this relevant. 

### To Reproduce

1. Create a time node an expose it to HA. Use a input_datetime an put the time to a time in the near furture (a few minutes in future but after restart of HA. Turn the switch off in HA. 
2. Restart HA
3. Check the switch entity in HA and re-deploy the time node in NR. It is supposed to be DISABLED
4. wait until the time you used before. The node should trigger even if DISBALED

### Expected behavior

When disabled it is not supposed to trigger

### Screenshots

![image](https://user-images.githubusercontent.com/53949754/119250342-d0ab1800-bb9f-11eb-8261-ac5845b22735.png)

![image](https://user-images.githubusercontent.com/53949754/119250351-e0c2f780-bb9f-11eb-8c22-666b4a50acb8.png)

![image](https://user-images.githubusercontent.com/53949754/119250370-f3d5c780-bb9f-11eb-8b6a-cc367401d578.png)

After restart
![image](https://user-images.githubusercontent.com/53949754/119250379-06500100-bba0-11eb-8965-776790ce93d4.png)

After re-deploy
![image](https://user-images.githubusercontent.com/53949754/119250400-1bc52b00-bba0-11eb-8dac-f97771ec5384.png)

### Example Flow

```
[{"id":"bc28b5d1.fe5588","type":"ha-time","z":"93117621.425f48","name":"Test new","server":"97a02c30.758c8","version":0,"exposeToHomeAssistant":true,"haConfig":[{"property":"name","value":"nr_alarm_test"},{"property":"icon","value":"mdi:alarm"}],"entityId":"input_datetime.test","property":"","offset":0,"offsetType":"num","offsetUnits":"minutes","randomOffset":false,"repeatDaily":true,"payload":"$entity().state","payloadType":"jsonata","debugenabled":false,"x":92,"y":1008,"wires":[["56c609bc.8dd778"]]},{"id":"56c609bc.8dd778","type":"debug","z":"93117621.425f48","name":"Alarm","active":true,"tosidebar":true,"console":false,"tostatus":true,"complete":"payload","targetType":"msg","statusVal":"payload","statusType":"auto","x":242,"y":1008,"wires":[]},{"id":"97a02c30.758c8","type":"server","name":"Home Assistant","version":1,"legacy":false,"addon":false,"rejectUnauthorizedCerts":true,"ha_boolean":"y|yes|true|on|home|open","connectionDelay":true,"cacheJson":true}]
```

### Environment (please complete the following information):

- Node Red Version: 1.3.4
- Node.js Version: 
- NR Home Assistant Plugin Version: 0.5.2
- Is Node Red running in Docker: Docker

### Other (please complete the following information):

- Have you searched previous issues for duplicates?: Yes

**Comment by DirkMu** [comment]
Tested with the latest release 0.33.0. Issue remains and can be reproduced.

**Comment by zachowj** [comment]
I was never able to reproduce this but I think the fix for #404 might have. Test it and see if it still around.

**Comment by DirkMu** [comment]
Looks good so far. Thanks!
